amethyst_utils/src/render.rs

```diff
@@ -0,0 +1,57 @@
+//! Module containing utility methods to easily create `Material` and `Texture` handles using a minimal amount of code.
```

Comment:
The whole module seems to be about `albedo` only materials, this should be indicated somehow. Possibly in

* module path or
* documentation (or both)

Reply:
Currently its like that, but it will have more eventually. I just ported code that I already had on hand.